apps/nextjs/src/components/board/mobile/mobile-board.module.css

```
@@ -0,0 +1,20 @@
+  grid-auto-rows: min(calc((100vw - 42px) / 2), 12rem);
+  gap: 10px;
+  padding-bottom: 10px;
+}
+
+.item {
+  position: relative;
+  min-width: 0;
+  min-height: 0;
+}
+
+.item > :global(.grid-stack-item-content) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)